test-tools/dynamic-aot-debug/README.md

```diff
@@ -0,0 +1,84 @@
+# Dynamic AOT Module Debugging
+
+This guide outlines the process for debugging WAMR AOT module compiled modules using dynamic AOT debugging features. Follow the steps below to set up and run your debugging environment.
+
+## How to use
+
+### 1. Enable Debug Configuration
+
+To enable dynamic AOT debugging, set the following
+compile macro switch:
+
+```
+WASM_ENABLE_AOT=1
+WASM_ENABLE_DYNAMIC_AOT_DEBUG=1
+```
```

Comment:
Had better mention `cmake -DWAMR_BUILD_DYNAMIC_AOT_DEBUG=1`

Reply:
done

Reply:
Had better be `cmake -DWAMR_BUILD_AOT=1 -DWAMR_BUILD_DYNAMIC_AOT_DEBUG=1`